### ISSUE TITLE
Not appearing in search engines

### ISSUE THREAD

**Issue by ConorMacBride** [sunpy/sunpy-sphinx-theme#132]
The sunpy main website, sunpy stable and latest docs all block robots. The following doesn't seem to have the intended effect:
https://github.com/sunpy/sunpy-sphinx-theme/blob/3c7125a876ec56a7ee5eb54d03c42770f8e173d9/sunpy_sphinx_theme/sunpy/layout.html#L17-L19

`to_be_indexed` seems to be defined here https://github.com/sunpy/sunpy-sphinx-theme/blob/3c7125a876ec56a7ee5eb54d03c42770f8e173d9/sunpy_sphinx_theme/sunpy/theme.conf#L21 however I cannot find any reference to `version_slug` anywhere. Does anyone know if and where `version_slug` is defined?

**Comment by Cadair** [comment]
Yeah this doesn't seem to be working. It would probably be better to reverse this logic anyway, so that if no `to_be_indexed` is defined all versions are indexed.

**Comment by nabobalis** [comment]
I will remove it and index everything.

**Comment by nabobalis** [comment]
Its been removed from the theme, now up to each package to deal with it.

**Comment by Cadair** [comment]
We should keep track of this, as we don't want to have to go and configure it in each sponsored package forever.

Thanks for the hot-fix though.
